docs/sources/installation/helm/reference.md

```diff
@@ -2660,6 +2660,15 @@ false
 			<td><pre lang="json">
 {}
 </pre>
+</td>
+		</tr>
+		<tr>
+			<td>rbac.namespaced</td>
+			<td>bool</td>
+			<td></td>
```

Comment:
Can we get a description please?

Reply:
I haven't changed that. For some reason, when running `documentation-helm-reference-check`, it introduces it. Looks it was omitted when RBAC cap[abilities was added. I will remove this block manually